CONTRIBUTING.md

```diff
@@ -1,35 +1,116 @@
+# Contributing
+
+Thanks for considering to contribute to this project. Please follow the below guidelines before you
+open up a pull request.
+
+## Nomenclature
+
+First a brief intro on some of the terminoligy and concepts we use to make sure the steps below make sense.
+
+_TODO: dmn, sttr, imtr etc. Consider moving this section to README.md instead_
+
+- permit, ('vergunning' in dutch). You might need a permit for an activity (activiteit).
+- topic, a type of doing (werkzaamheid in dutch). Could be multiple activities / permits in one topic. It corresponds with a configured checker with one or more permits.
+- sttr-file; sttr is an XML standard for dutch DSO legislation
+- xml to json; we convert sttr-XML to our custom json format for better performance
+- sttr-builder; the tool used to build sttr-files
+- check; the activity of checking whether you need a permit
+- visitor; the person performing a check
+- checker; the tool itself, intro, register lookups, a set of questions and conclusion
+
 ## Commiting
 
 We use [Conventional Commits](https://www.conventionalcommits.org).
 
-_Text to be extended..._
+## Updating permits
 
-## Adding a new topic with permits
+If you want to use the latest permit-configuration (XML files from the sttr-builder) follow these steps.
 
-Follow these steps if you want to add a new topic:
+- Make sure you have the api-key in your environment variables. `export STTR_BUILDER_API_KEY=somesecret`
+- Download the xml files and transform them to our json-format from `./apps/client` run `npm run generate`
```

Comment:
Running this gives me the following error: `sh: deno: command not found`, so maybe add some info about this